packages/react/src/components/Tag/Tag.js

```diff
@@ -9,7 +9,8 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import classNames from 'classnames';
 import { settings } from 'carbon-components';
-import { Close16 } from '@carbon/icons-react';
+import * as Icons from '@carbon/icons-react';
```

Comment:
I think it may be best to render icons using similar to the `renderIcon` prop found in other components

See Button and Tooltip for examples: 
https://github.com/carbon-design-system/carbon/blob/0049b31dfdfb54774eb49ab17f0df1b900cfa963/packages/react/src/components/Button/Button.js

https://github.com/carbon-design-system/carbon/blob/225f212a8f3e2bd86d4a23f374f205b25bf5b9f0/packages/react/src/components/Tooltip/Tooltip.js